docs/tikz/arch-single-standby.tex

```diff
@@ -1,32 +1,49 @@
 % Fix for: https://tex.stackexchange.com/a/315027/43228
 \RequirePackage{luatex85}
-\documentclass[border=10pt,12pt]{standalone}
+\documentclass[border=10pt,17pt]{standalone}
 
+\usepackage{cfr-lm}
 \usepackage{pgf}
 \usepackage{tikz}
 \usetikzlibrary{arrows,shapes,snakes,automata,backgrounds,petri}
 
 \begin{document}
 
+\definecolor{pbox}{HTML}{0078D4} % MS blue
+\definecolor{ptxt}{HTML}{FFFFFF} % white
+
+\definecolor{sbox}{HTML}{50E6FF} % MS cyan
+\definecolor{stxt}{HTML}{2F2F2F} % off-black
+
+\definecolor{mbox}{HTML}{9BF00B} % MS light green
+\definecolor{mtxt}{HTML}{2F2F2F} % off-black
+
+\definecolor{apbox}{HTML}{0078D4} % MS blue
+\definecolor{aptxt}{HTML}{2F2F2F} % off-black
+
+%% sans-serif fonts, large by default, and bold too
+\sffamily
+\sbweight
+\bfseries
+\large
```

Comment:
You can use `\input{some-shared-file.tex}` to include another `.tex` file. I think it's good to have some shared file(s) where we define these colors and import packages and define hese colors etc.

Reply:
Good point, done that way now!